distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFtp.java

```diff
@@ -305,6 +314,34 @@ public void testFtpsPutStream() throws Exception {
         verifyIngest(1, METACARD_TITLE);
     }
 
+    /**
+     * Upload a file via FTPS for ingest using streaming method and verify that the file is backed
```

Comment:
It seems odd to spread the testing of this plugin across 2 itests.  Would it be more appropriate to use the REST endpoint in `TestCatalog` so the feature is tested in a single itest?